src/main/java/com/safehiring/ems/api/EmployerAPI.java

```diff
@@ -30,4 +30,8 @@ public interface EmployerAPI {
 	@PutMapping(value = "/employmentoffer", produces = { "application/json" }, consumes = { "application/json" })
 	ResponseEntity<JobOfferResponse> updateEmploymentOffers(@RequestBody @Valid final JobOfferRequest jobOfferRequest);
 
+	@GetMapping(value = "/employmentoffer/{tin}")
+	ResponseEntity<List<JobOfferResponse>> getEmploymentOffersByTin(@PathVariable String tin);
```

Comment:
Don't need this endpoint for now. Already has this search employment offer API on line 27.